lib/profile_events_queue_handler.ts

```diff
@@ -29,25 +39,39 @@ const isProduction = process.env.NODE_ENV === "production";
 const publicApiUrl = getRequiredStringEnv("PUBLIC_API_URL");
 const publicApiKey = getRequiredStringEnv("PUBLIC_API_KEY");
 
+type WelcomeMessages = ReadonlyArray<
+  [(p: ExtendedProfile) => string, (p: ExtendedProfile) => string]
```

Comment:
perhaps, for consistency...

```ts
type WelcomeMessages = ReadonlyArray<(p: ExtendedProfile) => MessageContent>;
```

?

Reply:
done (I've used NewMessage that sets default values as well)